src/js/files/components/Manager.js

```diff
@@ -33,6 +33,11 @@ class FileManager extends React.Component {
         return <File key={id} id={id} />;
     };
 
+    validateExtensions = file => {
+        if (this.props.validationRegex.test(file.name)) return null;
+        return { code: "Invalid file type" };
+    };
```

Comment:
Just use unary here. If personally find this `if` without curly braces harder to grok.